### PR TITLE
Update Login UI + Country Code Picker

### PR DESCRIPTION
**Pull request by akexorcist**
* Update UI follows the original design (as it can be)
* Remove login toolbar title
* Add country code picker
* Phone number can only be signed number
* Show simple error message when start phone number auth failed

update-login-ui

### CODE REVIEW

File: Houseclub/build.gradle [hunk 1]
```diff
@@ -31,17 +31,20 @@ android {
...
-	implementation 'com.github.grishka.litex:viewpager2:1.0'
-	implementation 'com.github.grishka:appkit:c7854a3'
-	implementation 'com.squareup.okhttp3:okhttp:3.14.7'
-	api 'com.github.grishka.litex:annotation:1.0'
+    implementation 'com.github.grishka.litex:recyclerview:1.0'
+    implementation 'com.github.grishka.litex:swiperefreshlayout:1.0'
+    implementation 'com.github.grishka.litex:viewpager2:1.0'
+    implementation 'com.github.grishka:appkit:c7854a3'
+    implementation 'com.squareup.okhttp3:okhttp:3.14.7'
+    api 'com.github.grishka.litex:annotation:1.0'
```
Comment (grishka) [inline Houseclub/build.gradle]: Why did you change indents *everywhere*?

Comment (akexorcist) [reply Houseclub/build.gradle]: @grishka Sorry for my auto formatter 😆 

Let me revert the indents back for a moment

**Comment by staser** [comment]
Is it possible to make this selectable automatically based on user entry? (make country selection optional - e.g. one text field as it was before but with optional flag selector)

**Comment by grishka** [comment]
Oh wait. It ISN'T OPTIONAL?! I've always hated this kind of UI where they force you to go through this stupid list of countries instead of just entering "+7".

**Comment by grishka** [comment]
@akexorcist could you please make a new pull request that allows entering the country code manually as well as choosing from list?

**Comment by grishka** [comment]
(I did it myself)
